### DISCUSSION TITLE
Additional default directories to next lint

### DISCUSSION THREAD

**Discussion by mottox2** [vercel/next.js#25924] [Ideas]
### Describe the feature you'd like to request

In canary version, `next lint` has some default directories: `pages`, `components` and `lib`.
but my app use `src/pages` and `src/components` so I must create `next.config.js` and specify these directories.
I want these directories to be supported by default.

### Describe the solution you'd like

I would add `src/pages`, `src/components` and `src/lib` to default directories.

### Describe alternatives you've considered

I can use with `--dir` option to run build, but I would be happy to use it without options.

**Comment by housseindjirdeh** [comment]
Fixed here: #25952 :)

**Comment by lguima** [comment]
Hi there, I'm sorry if I couldn't post a question here in Discussion, I'm new to it in GitHub and there's no mention in Contribution guide, but this is very related.

I'm creating a new project using **Next.js 11.0.1** and it came with the `lint` script set to run `next lint` and I'm using a `src` folder same as @mottox2.

Even following the Next.js documentation, it seems it only works with the CLI command parameter, but not via `next.config.js` file (check out the screenshot below).

[Linting Custom Directories](https://nextjs.org/docs/basic-features/eslint#linting-custom-directories)
```
module.exports = {
  eslint: {
    dirs: ['pages', 'utils'], // Only run ESLint on the 'pages' and 'utils' directories during production builds (next build)
  },
}
```

I just would like to know if the changes in #25952 mentioned by @housseindjirdeh cover that?

Thank you!

![next-eslint](https://user-images.githubusercontent.com/1562097/124299314-abf97700-db33-11eb-8074-91c4031ca09b.PNG)

**Reply by housseindjirdeh** [reply]
@lguima #26401 covers exactly what you're looking for :)

In the meantime, you'll have to run `next lint --dir src`

**Reply by lguima** [reply]
@housseindjirdeh Awesome! Thank you!
